src/Behat/Testwork/Exception/TestworkException.php

```diff
@@ -10,11 +10,13 @@
 
 namespace Behat\Testwork\Exception;
 
+use Throwable;
+
 /**
  * All testwork exceptions implement this interface.
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  */
-interface TestworkException
+interface TestworkException extends Throwable
```

Comment:
This exception interface needs to extend the Throwable interface so that Psalm accepts that any exception which implements it can be thrown